src/donejs-carousel-plugin.js

```diff
@@ -172,7 +185,7 @@ export const ViewModel = DefineMap.extend({
 			this.activeSlideIndex--;
 		}
 		// move carousel to the new active slide
-		this.moveCarousel('active slide');
```

Comment:
Instead of passing params to viewModel functions, why not set a property for what type of carousel is being used and read from that?